lib/ansible/module_utils/cloudscale.py

```diff
@@ -35,7 +34,7 @@ def _get(self, api_call):
                                timeout=self._module.params['api_timeout'])
 
         if info['status'] == 200:
-            return json.loads(resp.read())
+            return self._module.from_json(to_text(resp.read()))
```

Comment:
Note: You probably want to use ```to_text(res.read(), errors='surrogate_or_strict')```.  That way if resp contains any undecodable bytes, it will throw an exception on python2 or translate them into something that can be round-tripped back into bytes on Python3.  Without that, the undecodable bytes will be turned into the unicode replacement character which could lead to subtle bugs.